src/lib/storage/vector_compression/vector_compression.hpp

```diff
@@ -16,7 +16,7 @@ namespace opossum {
  * Also known as null suppression and
  * zero suppression in the literature.
  */
-enum class VectorCompressionType : uint8_t { Invalid, FixedSizeByteAligned, SimdBp128 };
+enum class VectorCompressionType : uint8_t { FixedSizeByteAligned, SimdBp128 };
```

Comment:
Mmmh - I thought I saw "invalid" as an option somewhere before. Is it used somewhere else, too?

Reply:
Since I could remove it: Nope. An "Invalid" enum member is often a bad idea and can always be emulated with a std::optional<ThatEnum> if needed.